test/acceptance/reader.js

```diff
@@ -2,203 +2,215 @@
 var utils = require('../utils');
 var assert = require('assert');
 var nsq = require('../..');
+var uid = require('uid');
 
 describe('Reader', function(){
+  var topic = uid();
   beforeEach(function(done){
-    utils.deleteTopic('test', function(){
+    utils.deleteTopic(topic, function(){
+      topic = uid();
       done();
     });
   })
 
-  describe('with .nsqd addresses', function(){
-    it('should subscribe to messages', function(done){
-      var pub = nsq.writer();
-
-      var sub = nsq.reader({
-        topic: 'test',
-        channel: 'reader',
-        nsqd: ['0.0.0.0:4150']
-      });
-
-      pub.on('ready', function(){
-        pub.publish('test', 'something');
-      });
-
-      sub.once('message', function(msg){
-        msg.finish();
-        done();
-      });
+  afterEach(function(done){
```

Comment:
Yeah, good call. I moved them all to be `afterEach` since the topic name is created randomly now.